tests/acceptance/tests/Account/ForgotPassword.spec.ts

```diff
@@ -31,17 +31,17 @@ test ('As a customer, I can request a new password without existing customer ema
 }) => {
     await test.step('Navigate to login page and initiate password recovery', async () => {
         await ShopCustomer.goesTo(StorefrontAccountLogin.url());
-        await StorefrontAccountLogin.forgotPasswordLink.click();
+        await ShopCustomer.presses(StorefrontAccountLogin.forgotPasswordLink);
     });
 
     await test.step('Attempt to request password reset without entering an email', async () => {
-        await StorefrontAccountRecover.requestEmailButton.click();
+        await ShopCustomer.presses(StorefrontAccountRecover.requestEmailButton);
         await ShopCustomer.expects(StorefrontAccountRecover.passwordResetEmailSentMessage).not.toBeVisible();
     });
 
     await test.step('Request password reset with a non-existing email', async () => {
         await StorefrontAccountRecover.emailInput.fill('test-forgot-password-non-existing@email.net');
```

Comment:
```suggestion
        await ShopCustomer.fillsIn(StorefrontAccountRecover.emailInput, 'test-forgot-password-non-existing@email.net');
```
another one, replaced with .fillsIn() method as well!